pages/docs/evolution/change-review-process.md

```diff
@@ -47,3 +37,42 @@ Reviews that need a decision before the next committee meeting (e.g.
 because of a release deadline) can be done by the committee in
 email/chat. These type of reviews need to be requested by the Lead
 Kotlin Language Designer (currently Andrey Breslav).
+
+# Issue Template
+
+``` java
```

Comment:
Why ```java?

Reply:
Because the website doesn't support markdown as a language yet